vendor/blockchain_contracts/src/bin/calculate_offsets/ethereum/rfc003/compile_contract.rs

```diff
@@ -37,37 +27,27 @@ fn compile(file_path: &'static str) -> std::io::Result<()> {
                 .arg("--assemble")
                 .stdin(Stdio::piped())
                 .stdout(Stdio::piped())
+                .stderr(Stdio::null())
                 .spawn()?
         }
     };
 
-    let mut file = ::std::fs::File::open(file_path)?;
+    let mut file = ::std::fs::File::open(OsStr::new(&file_path))?;
 
     ::std::io::copy(&mut file, solc.stdin.as_mut().unwrap())?;
 
     let output = solc.wait_with_output()?;
     let stdout = String::from_utf8(output.stdout).unwrap();
-    let regex = Regex::new(r"\nBinary representation:\n(?P<hexcode>.+)\n").unwrap();
+    let regex = Regex::new(r"\nBinary representation:\n(?P<hexcode>.+)\n")?;
 
     let captures = regex
         .captures(stdout.as_str())
         .expect("Regex didn't match!");
 
-    let hexcode = captures.name("hexcode").unwrap();
-
-    let path = Path::new(file_path);
-    let folder = path.parent().unwrap().to_str().unwrap();
-    let folder = format!("{}/out", folder);
-    let file_name = path.file_name().unwrap().to_str().unwrap();
-
-    DirBuilder::new().recursive(true).create(&folder).unwrap();
-
-    let mut file = File::create(format!("{}/{}.hex", folder, file_name).as_str())?;
-    file.write_all(hexcode.as_str().as_bytes())?;
-
-    println!("cargo:rerun-if-changed={}", file_path);
+    let hexcode = captures.name("hexcode").ok_or(CaptureSolcBytecode)?;
```

Comment:
:wrench: Can we find a better name for `CaptureSolcBytecode`?
Also it is not a very common thing to import the variants of an enum, better refer to it as `Error::CaptureSolcBytecode`.